form.html

```diff
@@ -0,0 +1,159 @@
+              <input class="form__input-text" id="name" placeholder="Name" name="name" type='text'>
+            </div>
+            <div class="form__surname">
+              <label class="form__label-text" for="surname">Surname:</label>
+              <input class="form__input-text" id="surname" placeholder="Surname" name="surname" type='text'>
+            </div>
+            <div class="form__middle-name">
+              <label class="form__label-text" for="second-name">Middle name:</label>
+              <input class="form__input-text" id="second-name" placeholder="Middle name" name="second-name" type='text'>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <h2 class="form__section-header">Contact Information:</h2>
+            <div class="form__phone">
+              <label class="form__label-tel" for="phone">Contact number:</label>
+              <input class="form__input-tel" placeholder="Phone number" name="phone" type="tel">
+              <p class="form__paragraph--error">Error, double-check the number</p>
+            </div>
+            <div class="form__email">
+              <label class="form__label-email" for="email">Email:</label>
+              <input class="form__input-email" id="email" placeholder="user@domain.co.uk" name="email" type="email">
+            </div>
+          </section>
+
+          <section class="form__section">
+            <fieldset class="form__radio-list">
+              <legend class="form__radio-legend">Your General impression:</legend>
+              <ul class="form__radio-list">
+                <li class="form__radio-item">
+                  <div class="form__positive-answer">
+                    <input class="form__input-radio" id="answer-positive" name="answer" type="radio" value="rather-positive" checked/>
+                    <label class="form__label-radio" for="answer-positive">Rather positive</label>
+                  </div>
+                </li>
+                <li class="form__radio-item">
+                  <div class="form__negative-answer">
+                    <input class="form__input-radio" id="answer-negative" name="answer" type="radio" value="rather-negative"/>
+                    <label class="form__label-radio" for="answer-negative">Rather negative</label>
+                  </div>
+                </li>
+                <li class="form__radio-item">
+                  <div class="form__difficult-answer">
+                    <input class="form__input-radio" id="answer-difficult" name="answer" type="radio" value="difficult-answer"/>
+                    <label class="form__label-radio" for="answer-difficult">Difficult to answer</label>
+                  </div>
+                </li>
+              </ul>
+            </fieldset>
+          </section>
+
+          <section class="form__section">
+            <h2 class="form__header">Sights visited:</h2>
+            <ul class="form__checkbox-list">
```

Comment:
here you don't have field set, but you have UL again.

Reply:
So the UL shouldn't be used for this part of the designe?